src/pages/SettingsPage/UsersSettings/userDetail.jsx

```diff
@@ -280,6 +283,7 @@ const UserDetail = ({
         }).unwrap()
 
         toast.update(toastId.current, { render: `Updated user: ${user.name} ` })
+        dispatch(setUserData(data))
```

Comment:
Save a user that isn't you and it will update your user state. Only set when `user.self` is true.

Reply:
My mistake, fixed.